test/app/src/testApp.js

```diff
@@ -201,9 +206,13 @@ Object.assign(TestApp.prototype, {
   },
   logoutAndReload: function(event) {
     event && event.preventDefault();
-    this.logout().then(() => {
-      window.location.reload();
-    });
+    this.logout()
+      .catch(e => {
+        console.error('Error during signout: ', e);
```

Comment:
This will itself resolve the promise, so the window will reload in the then() clause below - is that what you intend?

Reply:
yes. we do not want errors to prevent reload